### PR TITLE
[WRAPPER] Fixed stack align on varargs (for #789)

### LINKED ISSUE #789: Running VS Code on RISC-V SBC
Electron-based apps seem still not working in general.

Logs for VS Code:
```
8521|SIGSEGV @0x3fd3629072 (strlen (/lib/riscv64-linux-gnu/libc.so.6)) (x64pc=0x2f0b53/???:"???", rsp=0x10b37c818), for accessing (nil) (code=1)
RAX:0x000000010b37ca40 RCX:0xffffffffffffffff RDX:0x0000000000000001 RBX:0x0000000000000100 
RSP:0x000000010b37c818 RBP:0x000000010b37c830 RSI:0x0000000000000100 RDI:0x000000010b37c910 
 R8:0x00000001029569a3  R9:0x000000010b37c8f0 R10:0x0000000000000017 R11:0x5555555555555555 
R12:0x00000001026e5303 R13:0x0000000000000000 R14:0x000000010b37c910 R15:0x000000010b37c910 
Segmentation fault
```

### PR DESCRIPTION
**Pull request by ksco**
Closes #789

### CODE REVIEW

**Comment by ptitSeb** [comment]
It broke one test on riscv it seems

**Comment by ptitSeb** [comment]
I'm pretty sure `(6 * 8 + 8 * 16)` is the right value

File: src/include/myalign.h [hunk 1]
```diff
@@ -132,6 +132,8 @@ typdef struct {
   }
 
 #elif defined(__loongarch64) || defined(__powerpc64__) || defined(__riscv)
+#undef X64_VA_MAX_XMM
+#define X64_VA_MAX_XMM  ((6*8)+(8*8))
```
[the remark this answers is not in the document]
Comment (ksco) [reply src/include/myalign.h:136]: The `reg_save_area` seems always 8 bytes wide, no matter what type.

**Comment by ptitSeb** [comment]
I think your 1st attempt was correct, with just 1 mystake: you should keep `area[fprs/8]`because `area` is a `uintptr_t` array, and so elements are 8-bytes in size, not 16 bytes (we don't take the whole XMM regs, just a double)

**Comment by ksco** [comment]
Oh yes, that's right. I got confused on the `area` and `fp_offset`, they're inconsistent in this situation.

**Pull request merged**